odc/geo/crs.py

```diff
@@ -140,14 +152,16 @@ def to_epsg(self) -> Optional[int]:
         """
         EPSG Code of the CRS or ``None``.
         """
+        if self._epsg == 0:
```

Comment:
```suggestion
        if self._epsg == EPSG_UNSET:
```

Reply:
I'm keeping it at zero for ease of `if self._epsg:` in `__eq__` and `.authority` functions, basically if `self._epsg` is `falsy` then it's either unknown or known to be absent.

Reply:
Okay, that is likely fine.